server/app/views/components/ButtonStyles.java

```diff
@@ -91,4 +91,12 @@ public final class ButtonStyles {
   // Just like CLEAR_WITH_ICON, but in dropdowns we want to remove the rounded corners.
   public static final String CLEAR_WITH_ICON_FOR_DROPDOWN =
       StyleUtils.removeStyles(CLEAR_WITH_ICON, "rounded-full");
+
+  public static final String LINK_STYLE =
+      StyleUtils.joinStyles(
+          StyleUtils.removeStyles(
```

Comment:
suggestion: I find this a little hard to reason about. Could we instead duplicate the contents of `BUTTON_BASE_OUTLINE_SEMIBOLD` here?

Also, `LINK_STYLE` is a little generic, could we name with something like `LINK_STYLED_BUTTON`

Reply:
suggestion (blocking):  Can we use one of the link styles that we already have so that our styles are consistent?  There's `LinkElement.java` and `ApplicantStyles.LINK`.  Seems like we could use one of the transparent buttons like we do in our other modals where we have "Continue to application" or "Continue without an account" (ButtonStyles.OUTLINED_TRANSPARENT).  @sijiayam could we use an existing style and mimic another modal?